src/extension.ts

```diff
@@ -541,23 +535,23 @@ function launchMetals(
       }
     );
 
-    registerCommand("metals.reset-choice-interactive", () => {
+    registerCommand(`metals.${ServerCommands.ResetChoice}-interactive`, () => {
       client.sendRequest(ExecuteCommandRequest.type, {
-        command: "reset-choice",
+        command: ServerCommands.ResetChoice,
         arguments: [],
       });
     });
 
-    registerCommand("metals.reset-choice", (args) => {
+    registerCommand(`metals.${ServerCommands.ResetChoice}`, (args) => {
```

Comment:
Can we actually just check if args is defined, otherwise use empty array? Something like:
`let realArgs = args || []`

We wouldn't need to define it twice and we could drop this weird `metals.${ServerCommands.ResetChoice}-interactive`

Reply:
Ah, good call. I actually changed this up a bit to instead have a default `[]` parameter. Which testing seems to work fine.